windows/vhba_hot_add_remove/create_new_file_read_write.yml

```diff
@@ -38,6 +38,27 @@
       - "test_line in get_file_content_result1.stdout"
     fail_msg: "The file content is '{{ get_file_content_result1.stdout }}', which should contain: '{{ test_line }}'"
 
+- name: "Set boot disk"
+  block:
+    - name: "Boot order change of hot-add disk to new SATA controller for OS with BIOS firmware"
+      ansible.builtin.debug:
+        msg:
+            - "The disk boot order may be changed after hot-add new disk which causes failure of powering on VM."
+            - "Specify the boot disk as workaround. "
+
+    - include_tasks: ../../common/vm_set_boot_options.yml
+      vars:
+        boot_hdd_name: "Hard disk 1"
```

Comment:
Another concern is we need to get VM boot disk name firstly before hotadd testing, then this 'Hard disk 1' is not hardcoded.

Reply:
Add the logic to get the boot disk name before vhba test.